spynnaker/pyNN/models/common/neuron_recorder.py

```diff
@@ -154,6 +184,8 @@ def _neurons_recording(self, variable, vertex_slice):
         :param ~pacman.model.graphs.common.Slice vertex_slice:
         :rtype: iterable(int)
         """
+        if variable in self.__events_per_core_variables:
+            return range(vertex_slice.lo_atom, vertex_slice.hi_atom + 1)
```

Comment:
Is this needed for these types of variable?  I think they won't need the data that this function generates (like per-timestep variables)

Reply:
Not needed any longer - moved into the _get_rewires function.